src/amo/components/Search/index.js

```diff
@@ -251,7 +256,7 @@ export class SearchBase extends React.Component<InternalProps> {
     if (errorHandler.hasError()) {
       log.warn(`Captured API Error: ${errorHandler.capturedError.messages}`);
 
-      if (errorHandler.capturedError.responseStatusCode === 404) {
+      if ([400, 404].includes(errorHandler.capturedError.responseStatusCode)) {
```

Comment:
We should document that because it might not be obvious in the future. I guess that's because the API rejects the invalid category?

Reply:
Yes, it does. It sends us a specific message about "invalid category", but I figured it made more sense to just 404 on any 400 than try to look for that specific message. Where would you recommend we document this? Just in a comment above this?

Reply:
the comment is perfect.